src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -157,6 +157,7 @@ private static JObject ToJson(StackItem item, Session session)
             {
                 Guid id = Guid.NewGuid();
                 session.Iterators.Add(id, iterator);
+                json["interface"] = nameof(IIterator);
```

Comment:
Could you, please, clarify, why do we need this line? We already have `JObject json = item.ToJson();` a few lines above, it creates `{"type": "InteropInterface" }`. After that, if we have an `id` inside this InteropInterface, then it's clear that it's Iterator hidden under the hood.

Are we planning to extend this API later?

Reply:
> Are we planning to extend this API later?

Yes.